### PR TITLE
feat (document): update upload document params

### PR DESCRIPTION
**Pull request by naufalihsank**
## Changes
- [update upload document params](https://github.com/jabardigitalservice/portal-jabar-cms/commit/81d2434b214f10054b99c738a97cf79ee2882f2b)
## Evidence
title: update upload document params
project: Portal Jabar
participants: @naufalihsank @Ibwedagama @rizfirman @agunghide @yoslie @doohanas

### CODE REVIEW

**Review by agunghide — APPROVED** [review]

File: src/pages/Profiles/CreateEditArchiveAndDocument.vue [hunk 1]
```diff
@@ -527,9 +528,15 @@ export default {
         this.documentUploadProgress = 0;
         this.documentUploadStatus = DOCUMENT_UPLOAD_STATUS.UPLOADING;
 
+        const uploadParams = {
+          domain: 'archives',
+          type: mime.extension(file.type),
+          is_set_alias_url: true,
```
Comment (Ibwedagama) [inline src/pages/Profiles/CreateEditArchiveAndDocument.vue:534]: ask

hello @naufalihsank can you explain what this params `is_set_alias_url` is for?

**Review by Ibwedagama — APPROVED** [review]

**Review by yoslie — APPROVED** [review]

File: src/pages/Profiles/CreateEditArchiveAndDocument.vue (hunk 1 above)
Comment (naufalihsank) [reply src/pages/Profiles/CreateEditArchiveAndDocument.vue:534]: so for now, the production of portal public url will display cloudfront url while user preview the document (uploaded by CMS user)

with this param, BE will change the url into file.jabarprov url 

cc @ayocodingit

Comment (Ibwedagama) [reply src/pages/Profiles/CreateEditArchiveAndDocument.vue:534]: I see, it this params only work for archive and document? @naufalihsank

**Review by rizfirman — APPROVED** [review]

**Review by doohanas — APPROVED** [review]

File: src/pages/Profiles/CreateEditArchiveAndDocument.vue (hunk 1 above)
Comment (naufalihsank) [reply src/pages/Profiles/CreateEditArchiveAndDocument.vue:534]: it should work in all modules, but for now @ayocodingit said that we implement it in document module first

**Pull request merged**